helpers/path.sh

```diff
@@ -18,6 +18,16 @@
 #                 firmware root path via $FIRMWARE_PATH
 #                 binary array via ${BINARIES[@]}
 
+check_path_valid() {
+  C_PATH="$1"
+  if [[ -n "$C_PATH" ]] && ( [[ "${C_PATH:0:2}" != "./" ]] && [[ "${C_PATH:0:2}" != "./" ]] && [[ "${C_PATH:0:3}" != "../" ]] ) ; then
+    print_output "[!] ""$C_PATH"" ""${C_PATH:0:1}"" ""${C_PATH:0:2}"" ""${C_PATH:0:3}"" "" is not a valid path in the context of emba" "no_log"
```

Comment:
I think we need to modify this check:

`  if [[ -n "$C_PATH" ]] && ( [[ "${C_PATH:0:1}" != "/" ]] && [[ "${C_PATH:0:2}" != "./" ]] && [[ "${C_PATH:0:2}" != "./" ]] && [[ "${C_PATH:0:3}" != "../" ]] ) ; then
`

Reply:
Yes, you are right - it's done.